packages/libaccessom2/package.py

```diff
@@ -13,9 +13,9 @@ class Libaccessom2(CMakePackage):
     homepage = "https://www.access-nri.org.au"
     git = "https://github.com/ACCESS-NRI/libaccessom2.git"
 
-    maintainers = ["harshula"]
+    maintainers("harshula")
 
-    version("master", branch="master")
+    version("access-om2", branch="master", preferred=True)
```

Comment:
Will there ever be any other version for `libaccessom2`? It feels odd to have an `"access-om2"` version for a package that is built expressly for ACCESS-OM2.

Reply:
I think we need to define at least one version. Since it is built only for ACCESS-OM2, the version name `access-om2` seems the most appropriate.

Reply:
Okay, I have no strong opinion. `"default"` would be another option, but that's probably not any clearer... I'll leave it up to you